packages/ngworker/lumberjack/src/lib/formatting/lumberjack-format-log.spec.ts

```diff
@@ -27,6 +27,10 @@ function parseFormattedLog(formattedLog: string) {
 
 describe(lumberjackFormatLog.name, () => {
   beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [LumberjackLogFactory, LumberjackTimeService],
```

Comment:
suggestion: Use `LumberjackModule.forRoot` instead of providing each dependency individually
```suggestion
      imports: [LumberjackModule.forRoot()],
```
(also add a relative import at the top of the file)